js/app/dashboard/components/context-menus/TasksContextMenu.js

```diff
@@ -177,13 +180,24 @@ export function getAvailableActionsForTasks(selectedTasks, unassignedTasks, link
   return actions
 }
 
+function usePrevious(old) {
```

Comment:
What does it do?

Reply:
i don't understand very well this code ( shame :( ) but it keeps in the ref the previous value of `selectedTasks`  so we can compare with the new value. like what we have before with `componentDidUpdate(prevProps, props)`.

Reply:
I see. Then maybe put it in a separate file so we can re-use it

Reply:
OK

Reply:
I think i didnt understand the code because the variable was badly named, it is indeed the "new value" which was called "old"

I moved it to utils.js for other usage